desafio-09/hoklifter/python/big_base.py

```diff
@@ -0,0 +1,78 @@
+'Big Base!'
+
+
+CHARS_VALUES = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
+
+
+def syntax_check(from_base, to_base, value):
+    'Checks syntax.'
```

Comment:
Mais detalhes ou remova. Esse comentário não tem muito propósito nessa forma.

Reply:
Não posso remover por causa do linter.

Reply:
Voce pode usar então algo mais util, tipo: `Check if the number is valid for a given base.`